plugins/ScheduledReports/lang/en.json

```diff
@@ -53,6 +53,8 @@
         "SuccessfullyUnsubscribed": "You have been successfully unsubscribed from the report %1$s.",
         "UnsubscribeFooter": "To unsubscribe from this report please follow this link: %1$s",
         "NoTokenProvided": "No token was provided in the URL",
-        "NoSubscriptionFound": "No subscription found. Maybe the report was already unsubscribed or removed."
+        "NoSubscriptionFound": "No subscription found. Maybe the report was already unsubscribed or removed.",
+        "EvolutionGraphsShowForEachInPeriod": "Evolution graphs show the evolution for %1$seach day%2$s in the last %3$s",
```

Comment:
should be 'show' since 'evolution graphs' is plural (in this case the option is for all evolution graphs in the report, which is why it's plural in the text)